pkg/metrics/policymetrics/policymetrics.go

```diff
@@ -63,6 +70,7 @@ func collect(ch chan<- prometheus.Metric) {
 	for _, policy := range list.Policies {
 		state := policy.State
 		counters[state]++
+		ch <- policyKernelMemory.MustMetric(float64(policy.KernelMemoryBytes), policy.Name)
```

Comment:
Does it make sense to report this metric for policies that are disabled or in error state?

Reply:
mmh I haven't tried, hopefully it should show up as zero if it's disabled since progs are unloaded and similarly for error 🤔

Reply:
Oh I tried and the memory state is actually not refreshed properly on disabled, it needs a fix, thanks for the remark.